release/api/v1alpha1/bundle_types.go

```diff
@@ -278,3 +279,10 @@ type Actions struct {
 type HaproxyBundle struct {
 	Image Image `json:"image"`
 }
+
+type SnowBundle struct {
+	Version    string   `json:"version"`
+	KubeProxy  Image    `json:"kubeProxy"`
+	Components Manifest `json:"components"`
+	Metadata   Manifest `json:"metadata"`
```

Comment:
same here i gues, `KubeVip` is needed to pass into the bootstrap script
https://github.com/aws/eks-anywhere/pull/1260/files#diff-a5e74cbe91995ae542581b95926f7d04f34a4fa830ade673b5d4616e2251bdfcR155